src/main/kotlin/racingcar/domain/RacingCar.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain
+
+class RacingCar(
+    var position: Int = 0
+) {
+    companion object {
+        private const val MOVE_POSSIBLE_NUMBER = 4;
+    }
+
+    fun move(input: Int) {
+        if (movePossible(input)) {
+            position++;
+        }
+    }
+
+    private fun movePossible(input: Int) = input >= MOVE_POSSIBLE_NUMBER
```

Comment:
```suggestion
    private fun movePossible(input: Int):Boolean = input >= MOVE_POSSIBLE_NUMBER
```

식을 쓰더라도 리턴 값을 넣어주는게 사이드 임팩트를 피할 수 있는 방법 중에 하나입니다 : )